geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl

```diff
@@ -1,6 +1,16 @@
 This file includes migration steps for each release of c2cgeoportal.
 
 
+Version 2.6.0
+=============
+
+Changes to apply
+----------------
+
+1. All the ngeo components will now be in the constants thats defined in the vars files in interfaces_config.
```

Comment:
Do we want to say
"All ngeo components can now be configured via variables in your project variables file, in the section interfaces_config. To ease future migration, you should remove any existing custom JavaScript code which is setting such variables."
?

Reply:
+1 :-)